backend/index.js

```diff
@@ -248,6 +252,8 @@ app.get('/api/bulk-appointments/view-model', controller.getBulkAppointmentsViewM
 app.post('/api/bulk-appointments', controller.addBulkAppointments)
 app.get('/bulk-appointments/csv-template', controller.bulkAppointmentsCsvTemplate)
 app.get('/api/missing-prisoners', controller.getMissingPrisoners)
+app.get('/api/get-alert-types', controller.getAlertTypes)
+app.post('/api/create-alert/:bookingId', handleErrors(controller.createAlert))
```

Comment:
Planning on moving the asyncHandler to this level to make testing the controller easier. For now, I've set the pattern.